### ISSUE TITLE
Allow for linking to existing IC/LBC files on GLADE if running on Derecho/Casper

### ISSUE THREAD

**Issue by jaredalee** [NCAR/wps_wrf_workflow#7]
## Describe the New Feature ##
Currently, there are scripts for downloading GFS or GEFS forecast files from AWS (download_[gfs|gefs]_from_aws.py). But when running this workflow on NSF NCAR HPC resources (i.e., Derecho or Casper), many IC/LBC datasets are stored locally on GLADE. Modify the workflow, adding new scripts and yaml environment variables as needed to enable linking to local files when available, and avoid unnecessary downloading of files from external repositories.

### Acceptance Testing ###
Start by linking to GFS forecast data stored locally on GLADE, as that's what's currently available at this time in this repo. When this workflow can successfully run ungrib, metgrid, and real on these steps (and optionally then also a very short WRF simulation), these changes are ready to accept.

### Time Estimate ###
1 day of work or less.

### Relevant Deadlines ###
Target completion by 18 Dec 2024. This needs to be done before enabling use of other datasets (e.g., HRRR, GFS FNL, etc.) for ICs/LBCs.

### New Parameters ###
May require a new parameter in the yaml file to denote whether to link to the data on GLADE or download from AWS (or elsewhere).

### Technical References ###
NONE

### Funding Source ###
Multiscale Workflow project.

### Assignee ###
- [x] Assign this Task to the appropriate person

### Labels ###
- [x] Select **component**
- [x] Select **priority**
- [x] Select **requestor(s)**

## New Feature Checklist ##
- [x] Complete the issue definition above.
- [x] Fork this repository or create a branch of **main**.
Branch name: `feature_<Issue Number>_<Description>`
- [x] Complete the development and test your changes.
- [x] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [x] Push local changes to GitHub.
- [x] Submit a pull request to merge into **main**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [x] Delete your fork or branch.
- [x] Close this issue.
